guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -148,7 +148,7 @@ A column can have the following attributes:
 </tr>
 </table>
 
-For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/etc` directory.
+For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc` directory.
```

Comment:
Hi @sanganinamrata. Please, add also:

"or `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`". 

There are two possible places where the mentioned XSD files can be located. It depends on how Magento is installed.

Reply:
@rogyar Sure, I'll do it and let you know.

Reply:
@rogyar I have updated.